ashlar/filters.py

```diff
@@ -162,35 +163,26 @@ class JsonBFilterBackend(BaseFilterBackend):
     """
     def filter_queryset(self, request, queryset, view):
         """ Filter by configured jsonb_filters on jsonb_filter_field """
+        lookup_name = 'jsonb'
         filter_field = getattr(view, 'jsonb_filter_field', None)
```

Comment:
Currently that's how I set it up - I figured we could add this logic back as necessary just so that this filter could be cleaner to reason about now